tests/components/google_assistant/test_http.py

```diff
@@ -144,7 +144,8 @@ async def test_call_homegraph_api_retry(hass, aioclient_mock, hass_storage):
         assert call[3] == MOCK_HEADER
 
 
-async def test_report_state(hass, aioclient_mock, hass_storage):
+@patch("homeassistant.helpers.start.async_at_start")
```

Comment:
Patching this out to avoid calling `AbstractConfig.async_initialize.sync_google` here:  https://github.com/home-assistant/core/blob/797b68b42dfb4e341792d25fcda91b5be6be012f/homeassistant/components/google_assistant/helpers.py#L115